course/layouts/learning_resource_types/taxonomy.html

```diff
@@ -1,9 +1,51 @@
-{{/* This is actually the "terms.html" template, more info here: https://discourse.gohugo.io/t/taxonomy-list-page-takes-a-wrong-template/20189 */}}
 {{ define "main" }}
 {{ partial "course_content.html" . }}
 
+{{- $numberOfResourcesLimit := 10 -}}
+{{- $showNoResourcesFoundMessage := true -}}
```

Comment:
We don't need this extra logic here with this `$showNoResourcesFoundMessage` flag.  All you need to do here is use an `{{ else }}` statement on the `{{ with ($.Site.GetPage (printf "/%s" $taxonomy)) }}`.  If the `with` statement evaluates as false or nil, it will fall back to the else statement.

Reply:
Okay, I see now that you're making that assessment based on if there are any pages with `content_type` equal to `resource` under the taxonomy and that makes sense.